src/components/header/Header.js

```diff
@@ -1,49 +1,91 @@
 import PropTypes from "prop-types";
 import React from 'react';
+import {connect} from "react-redux";
+import {bindActionCreators} from "redux";
 
 import './header.scss';
 import Logo from '../logo/Logo';
 import Search from '../search/Search';
 import Button from '../button/Button';
+import fetchData from '../../helpers/utils'
 
-const Header = ({callback}) => (
-  <header className="header">
-    <Logo />
-    <Search />
-    <Button
-      id='btnSearch'
-      title='Search'
-      disabled={false}
-      btnClass='btn--primary'
-      callback={callback}
-    />
-    <div>
-      <span>Searched by:</span>
-      <Button
-        id='btnTitle'
-        title='Title'
-        disabled={false}
-        btnClass='btn--primary'
-        callback={null}
-      />
-      <Button
-        id='btnGenre'
-        title='Genre'
-        disabled={false}
-        btnClass='btn--primary'
-        callback={null}
-      />
-    </div>
-  </header>
-);
+class Header extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      baseURL: 'https://reactjs-cdp.herokuapp.com/movies',
+      searchBy: '',
+    };
+  }
+
+  concatenateURL = () => {
+    const { baseURL, searchBy, search } = this.state;
```

Comment:
You can move this logic and baseUrl to fetchData helper.

Reply:
done